libs/datamodel/core/src/common/preview_features.rs

```diff
@@ -5,7 +5,7 @@ use std::fmt;
 macro_rules! features {
     ($( $variant:ident $(,)? ),*) => {
         #[enumflags2::bitflags]
-        #[repr(u32)]
+        #[repr(u64)]
```

Comment:
I had to change this because `DenoDeploy` is the 33rd flag and I saw the error `Not enough bits for 33 flags` with `repr(u32)`